examples/gno.land/p/demo/gnolotto/gnolotto.gno

```diff
@@ -0,0 +1,100 @@
+package gnolotto
+
+import (
+	"std"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Ticket struct {
+	Numbers []int       // Holds the selected numbers for the lottery ticket
+	Owner   std.Address // Address of the ticket owner
+}
+
+type Lottery struct {
+	Tickets        []Ticket  // All tickets in the lottery
+	WinningNumbers []int     // Winning numbers after the draw
+	DrawTime       time.Time // Time of the draw
+	PrizePool      int64     // Total prize pool amount
+}
+
+// Intializes a new lottery instance with a specified draw time and prize pool
+func NewLottery(drawTime time.Time, prizePool int64) *Lottery {
+	return &Lottery{
+		DrawTime:  drawTime,
+		PrizePool: prizePool,
+		Tickets:   make([]Ticket, 0),
+	}
+}
+
+// Adds a new ticket to the lottery
+func (l *Lottery) AddTicket(numbers []int, owner std.Address) {
+	l.Tickets = append(l.Tickets, Ticket{Numbers: numbers, Owner: owner})
+}
+
+// Conducts the draw by generating 5 random numbers between 1 and 15
+func (l *Lottery) Draw() {
+	var blockHeight int64 = std.GetHeight()
+
+	l.WinningNumbers = nil
+	numbersMap := make(map[int]bool)
+
+	// Add variability to the pseudo-random number generation
+	var variabilityFactor int64 = 1
+
+	for len(l.WinningNumbers) < 5 {
```

Comment:
You can define the 5 as a constant, since it is used throughout the app

Reply:
👍